st2tests/st2tests/policies/meta/BUILD

```diff
@@ -0,0 +1,4 @@
+resources(
+    name="policies_meta",
+    sources=["*.yaml"],
+)
```

Comment:
Adding this BUILD file broke a test that was checking directory contents. So, I adjusted the test to account for it.